custom-connectors/M365Lighthouse/apiProperties.json

```diff
@@ -0,0 +1,83 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oAuthSetting",
```

Comment:
oAuthSetting is case sensitive, please refer below schema and modify. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json#L96